src/components/panel/demoGroups/style.global.css

```diff
@@ -0,0 +1,80 @@
+.menu-panel-container {
+  pointer-events: all;
```

Comment:
@ErinCoughlan In order to have pointer events reach anything beyond the panel, isn't it necessary to include `propagateContainerEvents: true`? And, in doing so, you allow `pointer-events` to reach through the panel as well. This will result in a menu, such as in the example, that is unable to be interacted with. To catch `pointer-events` only on the panel itself, I added the `pointer-events` CSS class.